torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1165,10 +1179,6 @@ def method_tests():
         ('atan2', (S, S, S), ((S,),), 'broadcast_rhs'),
         ('atan2', (S,), ((S, S, S),), 'broadcast_lhs'),
         ('atan2', (S, 1, S), ((S, S),), 'broadcast_all'),
-        ('reciprocal', torch.rand(S, S, S) + 0.1, NO_ARGS, '', (True,)),
```

Comment:
Removing these is great. Take a look at other tests for reciprocal, too, to see if they can be removed. For example:

https://github.com/pytorch/pytorch/blob/7d406b4a0751afdc2bd20d7be0920986178b41ae/test/test_unary_ufuncs.py#L910